src/keycloak/zarf.yaml

```diff
@@ -7,6 +7,9 @@ variables:
   - name: DOMAIN
     description: "Cluster domain"
     default: "uds.dev"
+  - name: INSECURE_ADMIN_PASSWORD_GENERATION
+    description: "Generate an insecure admin password for dev/test"
+    default: "false"
```

Comment:
todo: test alternative methods here to avoid exposing this with a zarf var

Reply:
Resolved this by using a port forward and tasking to automate admin user creation.